databases/sql-2.md

```diff
@@ -45,24 +45,26 @@ Here's a sample schema:
         - which has an integer field named: `user_id`
         - and a text field named: `content`
 
-2.  Enter the SQLite command line program using the terminal commad `sqlite3 twitter.sqlite3`. (This will save your database to a file called `twitter.sqlite3`). Once there, use SQL statements to create two tables according to the spec for the two tables in step 1.
+2. Save your database by using the `command pg_dump twitter > twitter.psql`. (This will save your database to a file called `twitter.psql`.)
 
-3.  Once you have your two tables set up, compare your table set-up with another apprentice's.
+3. Now enter the PostgreSQL command line program using the terminal command `psql`. Once there, use SQL statements to create two tables according to the spec for the two tables in step 1.
 
-4.  Add sample data to the tables yourself (make up some users and posts).
+4.  Once you have your two tables set up, compare your table set-up with another apprentice's.
 
-5.  Try writing queries that get data such as:
+5.  Add sample data to the tables yourself (make up some users and posts).
+
+6.  Try writing queries that get data such as:
 
     - All the tweets by a given user
     - The 10 most recent tweets by any user
     - Use a join to get a user's info along with their tweets
     - Make up your own!
 
-6.  Now let's perform a database migration. We will add a new column to an existing table in our database. Add a text field named: `bio` to your existing `users` table!
+7.  Now let's perform a database migration. We will add a new column to an existing table in our database. Add a text field named: `bio` to your existing `users` table!
 
-7.  Add some sample data to your new `bio` fields, but not for every user (leave some of their bios blank).
+8.  Add some sample data to your new `bio` fields, but not for every user (leave some of their bios blank).
```

Comment:
[Optional] It's okay to leave them but for future reference, when making these numbered list items in markdown, it's actually better to make them all `1.` because that way they will be auto-numbered.